<launch>

  <!-- The laser odometry plugin type. -->
  <arg name="laser_odometry_type"/>

  <!-- A config file for the laser odometry plugin. -->
  <arg name="node_config_file" default="$(find laser_odometry)/config/laser_odometry_node_default.yaml"/>

  <!-- The input topic. -->
  <arg name="topic_in" default="~topic_in"/>

  <node pkg="laser_odometry" type="laser_odometry_node" name="laser_odometry_node" output="screen">
    <param name="laser_odometry_type" value="$(arg laser_odometry_type)" />
    <rosparam file="$(arg node_config_file)" command="load" />
    <remap from="~topic_in" to="$(arg topic_in)" />
  </node>

</launch>
